app.py

```diff
@@ -186,29 +202,53 @@ def update_figure(hover_data, redundancy_check, feature_solution_data, data_key)
             color = 'Redundancy'
             hover_cols['Redundancy'] = False
 
-    opacity = 1.0
-    mlw = 0
+    # If displaying a solution, show selected feature as solid and non-selected
+    # features as transparent
+    opacity = np.repeat(1.0, len(df))
+    if data.n < 50:
+        mlw = np.repeat(1, len(df))
+    else:
+        mlw = np.repeat(0, len(df))
```

Comment:
Take this one or leave it, I just find this easier to read, but it's just personal preference.

```suggestion
    mlw = np.repeat(1 if data.n < 50 else 0, len(df))
```

Or you should take this one step further with 
```suggestion
    mlw = np.repeat(int(data.n < 50), len(df))
```